### DISCUSSION TITLE
How to use Github actions for multiplatform packaging?

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#501] [Q&A]
Can anyone guide me on how to use Github actions for multiplatform packaging?
I have been using Appveyor till now(which works fine), but want to also try GithubActions. 
Thanks in advance.

**Comment by liudonghua123** [comment]
you can look at my github action on https://github.com/liudonghua123/statistics_app/blob/flet/.github/workflows/main.yaml.

**Reply by ndonkoHenri** [reply]
It seems pretty complex though. 😕
